users/migrations/0011_justfixuser_preferred_first_name.py

```diff
@@ -6,13 +6,18 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('users', '0010_alter_justfixuser_first_name'),
+        ("users", "0010_alter_justfixuser_first_name"),
     ]
 
     operations = [
         migrations.AddField(
-            model_name='justfixuser',
-            name='preferred_first_name',
-            field=models.CharField(blank=True, help_text='The first name Justfix will call the user by. Optional.  May be different from their legal first name.', max_length=150, verbose_name='Preferred first name'),
+            model_name="justfixuser",
+            name="preferred_first_name",
+            field=models.CharField(
+                blank=True,
+                help_text="The first name Justfix will call the user by. Optional.  May be different from their legal first name.",
+                max_length=150,
+                verbose_name="Preferred first name",
+            ),
```

Comment:
I was confused by this at first, but I'm guessing what happened is that you saved this file from VSCode, which immediately ran Black on it.  Normally we don't check for Black formatting on these files b/c they're auto-generated by Django, but it's hard/annoying to also tell VSCode to ignore them when auto-formatting on save.

Anyways as far as I can tell this is purely Black at work, and contains no semantic changes, but let me know if I'm mistaken!

Reply:
Ah yes indeed, this was Black at work. I'll revert - I think it causes more confusion than it's worth.